pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/PostMedicalProviderRouteTest.java

```diff
@@ -267,7 +271,7 @@ private static final class TestMedicalProviderData {
         public static final String PHYSICIAN_NAME = "House MD";
         public static final String CITY = "West Windsor Township";
         public static final String STATE = "New Jersey";
-
+        public static final String COUNTRY = "United States";
```

Comment:
Using the 2 letter codes would be preferred to using the readable name in the request payload.

Reply:
After autocompletion, suggested countries are fully written, populated data of country names in country_address_info table is also written as a full country name. When post request is being sent country names are written fully and we do search for corresponding ids in  country_address_info via full name, therefore if we use ISO 3166-1 alpha-2 standard, we might mess up this test.

Reply:
Where are the autocompletion details coming from?

Reply:
It comes from : institution, city, country_subnational_division, country_address_info tables.